build/sage_bootstrap/compat/__init__.py

```diff
@@ -28,3 +28,11 @@
     from StringIO import StringIO
 except ImportError:
     from io import StringIO
+
+
+try:
+    # Python >= 3.3
```

Comment:
3.3?

Reply:
yes

Reply:
I am a bit confused. If it is available from python 3.3, surely you will only get to the exception for python <3.3 not 3.13. And we never supported python 3.2 IIRC.

Reply:
sage-bootstrap still supports python 2